spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -229,9 +241,27 @@
           flow_path: 'standard',
         )
 
-        expect(@irs_attempts_api_tracker).not_to receive(:track_event).with(
+        expect(@irs_attempts_api_tracker).to receive(:track_event).with(
+          :idv_document_upload_rate_limited,
+        )
+
+        # This is the last upload which triggers the rate limit, apparently.
+        # I do find this moderately confusing.
```

Comment:
Someone want to check my logic here? I wasn't expecting both events initially, but that seems to be what happens with analytics events.

Want to make sure I'm enshrining a new bug in tests.

Reply:
I think there's the attempt, and then that triggers the limit? I think that's fine